### DISCUSSION TITLE
Feature Request: Preserve Underlying Transparency

### DISCUSSION THREAD

**Discussion by jhughes696** [airbnb/lottie-ios#1920] [General]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when I need use multiple animated layers as a source for an Alpha Matte. So someday, it would be amazing to have support for **Preserve Underlying Transparency**.  It takes the combined alpha channel of all the layers below and clips the selected layer inside this "composite" Alpha channel of every layer below. 

**Describe the solution you'd like**
Add support for After Effects native feature "**Preserve Underlying Transparency**"

**Describe alternatives you've considered**
Sometimes (in the case of this character animation) you need your selected layer and the AlphaMatte source layers to be parented to the same null. In this example, I need the shadow layer AND the character's layers to be parented to the same master nulls for the Body and the Scooter. Usually, I would precompose the character's layers to create an animated AlphaMatte; but pre-composing the animated source layers means they can't be parented to the Body null or Scooter null in the main comp. So it would be more efficient if I could keep the shadow layer and character's layers in the main comp, place the shadow layer on top, and turn on **Preserve Underlying Transparency** in order to matte the shadow inside the character's layers below– all whilst preserving the parent chains to the Body null and Scooter null: 

![_050_Oscar_LOTTIE_1](https://user-images.githubusercontent.com/50207823/213238367-9aa40474-9c32-4e3a-a0a1-3b831a09f6d4.gif)
<img width="436" alt="shadow" src="https://user-images.githubusercontent.com/50207823/213239132-a85d1202-c7f5-4463-ae74-ee60bd89905d.png">

**Additional context**
Here is another example along with an [AE test file](https://www.dropbox.com/s/4dgnkgqktiygm6o/preserveUnderlyingTransparency.aep?dl=0): 

![preserveUnderlyingTransparency_GIF](https://user-images.githubusercontent.com/50207823/213208268-4a058541-3b83-4ced-8fec-4748b4c14568.gif)
<img width="446" alt="preserveUnderlyingTransparency" src="https://user-images.githubusercontent.com/50207823/213208407-7aca6800-2f62-4f6c-a9d4-c88be2eaa7e4.png">
<img width="723" alt="preservUnderlyingTest" src="https://user-images.githubusercontent.com/50207823/213208412-54423d16-fb4a-4970-b4b6-c26ec98e8d02.png">

**Comment by calda** [comment]
Thanks for the info -- I'm going to convert this to a discussion for now since this is a brand new feature that doesn't exist on any platform yet
